### PR TITLE
Presskit page #1683

### PR DESCRIPTION
**Pull request by olafghanizadeh**
PR for this bounty. It's quite far along as of now.

TODO:
- [x] Fix remaining alignment bugs
- [ ] Add translation tags for text
- [x] Test responsiveness

### CODE REVIEW

**Comment by olafghanizadeh** [comment]
> responsiveness messed up here http://bits.owocki.com/3p031G3K2a3H/Screen%20Shot%202018-07-12%20at%2010.22.54%20PM.png

Damn, what resolution do you use. 

Yeah, scaling SVG's responsively is a little tricky sometimes. I'll work on it.

**Comment by olafghanizadeh** [comment]
@owocki are those also supposed to be added to the "Illustrations" section?

**Comment by owocki** [comment]
@olafg i dont think so, i was just answering this question from you:

> can we add links to these images (and links everywhere)

**Comment by thelostone-mc** [comment]
@olafghanizadeh good for review ? :P

**Comment by olafghanizadeh** [comment]
@thelostone-mc should be good for a review yeah

File: app/retail/templates/presskit.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <div class="container-fluid header">
+      {% include 'shared/nav.html' %} {% include 'shared/minihero.html' with h1='Press Kit' %}
+    </div>
+    <div class="presskit">
+      <section class="presskit_section py-5">
+        <div class="container">
+          <div class="row my-5">
+            <h2 class="col-12 my-3">{% trans "Gitcoin Press Kit" %}</h2>
+            <div class="col-sm-12 col-md-6">
+              <p>{% blocktrans %} Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam vestibulum leo sed interdum ultrices. Fusce suscipit auctor urna ac ultrices. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nam molestie
```
Comment (mbeacom) [inline app/retail/templates/presskit.html]: Can you drop the `blocktrans` for the Latin placeholder text?

File: app/retail/templates/presskit/colors.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<section class="presskit_section py-5">
+  <div class="container">
+    <div class="row">
+      <h2 class="col-12 my-3 mb-5">{% trans "Colors" %}</h2>
+      <div class="col-lg-4">
+        <p>{% blocktrans %}Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam vestibulum leo sed interdum ultrices. Fusce suscipit auctor urna ac ultrices. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nam molestie
```
Comment (mbeacom) [inline app/retail/templates/presskit/colors.html]: We can drop `blocktrans` here.

File: app/retail/templates/presskit.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+{% include 'shared/head.html' %}
+<link rel="stylesheet" href="{% static "v2/css/presskit.css" %}">
```
Comment (thelostone-mc) [inline app/retail/templates/presskit.html]: Could we indent the above two lines by 1 tab ?

File: app/retail/templates/presskit/logo.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+<section class="presskit_section py-5">
+  <div class="container">
+    <div class="row my-5">
+      <h2 class="col-12 my-3">{% trans "Logo" %}</h2>
+      <div class="col-lg-4 mb-3">
+        <p>{% blocktrans %} Gitcoin is a mission and value-oriented
+distributed movement intended to drive change and grow open source. The Gitcoin logo may be used per the following guidelines.  {% endblocktrans %}</p>
+<p>{% blocktrans %}Do not alter the Gitcoin logo or mark in any way, including changing the colors, angle, or dimension.{% endblocktrans %}
+</p>
+<p>{% blocktrans %}Ensure that the Gitcoin mark and logo type stands out from surrounding text or is legible.{% endblocktrans %}</p>
```
Comment (thelostone-mc) [inline app/retail/templates/presskit/logo.html]: re-indent needed

File: app/retail/templates/presskit/resources.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            <img src="{% static "v2/images/presskit/ZipFile.svg" %}">
+          </div>
+          <div class="col-md-8 px-3">
+            <div class="card-block px-3 text-center text-md-left">
+              <h4 class="card-title text-center text-md-left">{% trans "Press Kit" %}</h4>
+              <p class="card-text">{% trans "dropbox.com, 84 MB" %}</p>
+              <a class="my-3 font-weight-bold" target="_blank" rel="noopener noreferrer" href="https://www.dropbox.com/sh/bsjzbu0li2z0kr1/AACKgnQC3g6m52huYI3Gx3Ega?dl=0">Download Press Kit</a>
+            </div>
+          </div>
+
```
Comment (thelostone-mc) [inline app/retail/templates/presskit/resources.html]: let's remove the extra line

**Comment by thelostone-mc** [comment]
@olafghanizadeh left a few comments ! Could you resolve those ?

**Comment by PixelantDesign** [comment]
Looking good!

**Comment by thelostone-mc** [comment]
@olafghanizadeh

x1



-  mind squashing the commits ?
- `.color-card > .card-header` -> could we set the height to 10rem to make it feel more along the lines of a square?
- Logos seem to be going on the uneven size on 15 inch screens and lower ( as shown in the gif )

**Comment by thelostone-mc** [comment]
squashed and made it into once nice commit  👍

**Comment by owocki** [comment]
@PixelantDesign @thelostone-mc theres a bunch of Lorem Ipsum  in the text of this page.  i dont think we should link to gitcoin.co/presskit until that text is updated.

@PixelantDesign as brand owner, i feel like youre best positioned to write some copy for [this page](http://bits.owocki.com/0g2A382r0V2j/screencapture-localhost-8000-presskit-2018-07-20-12_36_34.png).  mind taking a pass?

**Comment by PixelantDesign** [comment]
Yes I'll write some copy for this.

**Comment by owocki** [comment]
hey @PixelantDesign https://gitcoin.co/presskit is live with the Lorem Ipsum still. i think this might have gotten lost in your post-travel inbox, so just want to make sure its on your radar still!

**Comment by PixelantDesign** [comment]
Copy updated! Thanks @mbeacom

**Comment by owocki** [comment]
now that this page has updated copy.. should we be linking it anywhere?

**Comment by PixelantDesign** [comment]
Ideally the press section of gitcoin.co would link to a press page, and somewhere on the press page would be this press kit! :)

We could also put it on the bottom of the About page.

**Pull request merged**